pkg/network/tags_linux.go

```diff
@@ -25,5 +31,5 @@ func GetStaticTags(staticTags uint64) (tags []string) {
 }
 
 func IsTLSTag(staticTags uint64) bool {
-	return staticTags&tlsTagsMask > 0
+	return staticTags&ConnTagTLS > 0
```

Comment:
you should use the old method as well
the `ConnTagTLS` is being set if we are capturing the handshake, while the other tags can appear if we have uprobes although we didn't capture the tls handshake

Reply:
TagTLS don't detect handshake only but I can add all tag to be in this case to be on the safe side